extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -38,7 +38,7 @@ tags: {  # <amp-date-countdown>
   }
   attrs: {
     name: "end-date"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds','timeleft-ms']"
```

Comment:
nit: please alphabetize, `timeleft-ms` should be after `end-date`. same with others below